### PR TITLE
Update localization

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

This PR update localization section in swagger

# Reference

https://jira.kisio.org/browse/BOT-1388

https://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/feature_BOT-1338_localization/documentation/swagger.yml

### CODE REVIEW

File: documentation/swagger.yml [hunk 1]
```diff
@@ -1919,10 +1919,37 @@
...
+                items:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                      description: Type of code
+                    value:
+                      type: string
+                      description: Value of code
+              coords:
```
Comment (Fahmus) [inline documentation/swagger.yml]:
```suggestion
              coord:
```

**Comment by Fahmus** [comment]
You must update GET ​/disruptions​/{id}/history also :)

File: chaos/fields.py [hunk 2]
```diff
@@ -218,12 +217,19 @@ def output(self, key, obj):
         return self._clean_localizations(to_return)
 
     def _clean_localizations(self, localizations):
+        filtred_localizations = []
```
Comment (pthegner) [inline chaos/fields.py]: i think it's more filtered_localizations.

File: chaos/fields.py [hunk 3]
```diff
@@ -218,12 +217,19 @@ def output(self, key, obj):
...
         for localization in localizations:
-            if localization.has_key('timezone'):
-                del localization['timezone']
-            if localization.has_key('links'):
-                del localization['links']
-        return localizations
+            filtred_localization = {
```
Comment (pthegner) [inline chaos/fields.py]: maybe you should use : 
filtered_localization = { key: localization[key] for key in localization.keys() if key in ('id','label','name','type','codes','coord')}

**Pull request merged**